freedomofpress/securedrop
# Remove "Refresh codename" feature

## Status
Ready for review

## Description

Removes the "Refresh" icon from the page that presents a source with a seven word codename. In practice, this feature has potential for user confusion, as it introduces complexity to a critical step of the submission process while delivering minimal end user value.

One key motivation for this feature has been the occasional generation of problematic words in codenames. We have already iterated over the wordlists several times, and will continue to do so.

## Testing

1. Log into the Source Interface
2. Verify that there no longer is an icon to refresh the codename

## Deployment

Note that this PR removes an image from the package, which is also removed from the AppArmor allow-list.

## Checklist

### If you made changes to the server application code:


### If you made changes to `securedrop-admin`:


### If you made changes to the system configuration:


### If you added or removed a file deployed with the application:


### If you made non-trivial code changes:


Choose one of the following:
